Update issue templates (bugs/features) (#1341)
nest/nest-simulator: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Create a report to help us improve
title: ''
labels: ''
assignees: ''

---

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. (Minimal) reproducing example
2. Used parameters
3. Command to run
4. How to see error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop/Environment (please complete the following information):**
 - OS: [e.g. iOS, Ubuntu-18.04]
 - Browser: [if applicable, e.g. chrome, safari]

>  - Shell: [e.g. bash, zshshell]

Reviewer: ```suggestion
 - Shell: [e.g. bash, zsh]
```

 - Python-Version: [e.g. Python 3.6]
 - NEST-Version: [e.g. nest-2.18, or git hash]
 - Installation: [e.g. conda packet, apt, with/out MPI, ...]

>    (in case of manual compile from sources provide the

Reviewer: ```suggestion
   (in case you compiled the source code manually, provide the
```
